config/routes.rb

```diff
@@ -18,15 +17,14 @@
     get 'driver' => 'devise/sessions#new', :as => :new_driver_session
     post 'driver' => 'devise/sessions#create', :as => :driver_session
     delete 'driver' => 'devise/sessions#destroy', :as => :destroy_driver_session
-    get 'drivers/profile' => 'drivers/profile', :as => :driver_root
+    get 'drivers/orders', :as => :driver_root
```

Comment:
[Use the Ruby 1.9 hash literal syntax when your hash keys are symbols.](https://github.com/bbatsov/ruby-style-guide#hash-literals)